runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowRunner.java

```diff
@@ -769,7 +769,28 @@ public DataflowPipelineJob run(Pipeline pipeline) {
               dataflowOptions.getPathValidator().verifyPath(options.getGcpTempLocation()));
     }
     newJob.getEnvironment().setDataset(options.getTempDatasetId());
-    newJob.getEnvironment().setExperiments(options.getExperiments());
+
+    // min_cpu_platform flag is available in PipelineOption. It however requires to be triggered
```

Comment:
Small suggestion: a single comment here might suffice. Say:

"// Represent the minCpuPlatform pipeline option as an experiment, if not already present."

Reply:
Done.